### PR TITLE
I18n(es): Updating some pages translation

### PR DESCRIPTION
**Pull request by DavidEGiraldo**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updating `error-reference.mdx`, `local-image-used-wrongly.mdx` and `expected-image.mdx` pages translation from this commit: https://github.com/withastro/docs/commit/106bdf93ebc68c885850e043a2310779011d6b1e.

### CODE REVIEW

File: src/content/docs/es/reference/errors/expected-image.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ i18nReady: true
 githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
 ---
 
-> **ExpectedImage**: Se esperaba que la propiedad `src` fuera una imagen importada como ESM o una cadena de texto con la ruta de una imagen remota. Se recibieron `OPTIONS`.
+> **ExpectedImage**: Se esperaba que la propiedad `src` de `getImage` o `<Image />` fuera o bien una imagen importada como ESM o una cadena de texto con la ruta de una imagen remota. Se recibió `SRC` (tipo: `TYPEOF_OPTIONS`). Opciones completamente serializadas recibidas: `FULL_OPTIONS`.
```
Comment (Waxer59) [inline src/content/docs/es/reference/errors/expected-image.mdx:7]:
```suggestion
> **ExpectedImage**: Se esperaba que la propiedad `src` de `getImage` o `<Image />` fuera o bien una imagen importada como ESM o una cadena de texto con la ruta de una imagen remota. Se recibió `SRC` (tipo: `TYPEOF_OPTIONS`). Se recibieron todas las opciones serializadas: `FULL_OPTIONS`.
```

File: src/content/docs/es/reference/errors/local-image-used-wrongly.mdx [hunk 2]
```diff
@@ -18,8 +20,14 @@ import myImage from "../my_image.png";
 <!-- BIEN: `src` es la imagen completa importada. -->
 <Image src={myImage} alt="Cool image" />
 
+<!-- BIEN: `src` es una URL. -->
+<Image src="https://example.com/my_image.png" alt="Cool image" />
```
Comment (Waxer59) [inline src/content/docs/es/reference/errors/local-image-used-wrongly.mdx:24]:
```suggestion
<Image src="https://example.com/my_image.png" alt="Imagen genial" />
```

File: src/content/docs/es/reference/errors/local-image-used-wrongly.mdx [hunk 3]
```diff
@@ -18,8 +20,14 @@ import myImage from "../my_image.png";
...
+
 <!-- MAL: `src` es la ruta `src` de la imagen en lugar de la imagen completa. -->
 <Image src={myImage.src} alt="Cool image" />
+
+<!-- MAL: `src` es una cadena de caracteres con una ruta de archivo. -->
+<Image src="../my_image.png" alt="Cool image" />
```
Comment (Waxer59) [inline src/content/docs/es/reference/errors/local-image-used-wrongly.mdx:30]:
```suggestion
<Image src="../my_image.png" alt="Imagen genial" />
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Good work, and welcome to Team Docs, @DavidEGiraldo!

**Pull request merged**